content/influxdb/v2.0/api-guide/api_intro.md

```diff
@@ -31,11 +31,12 @@ an [InfluxDB Cloud account](/influxdb/cloud/sign-up).
 
 ### Authentication
 
-Before diving into the API, use the InfluxDB UI to
+InfluxDB uses [API tokens](/influxdb/v2.0/security/tokens/) to authorize API requests.
+
+1. Before diving into the API, use the InfluxDB UI to
```

Comment:
Less colloquial but still expressive? Just an idea.

```suggestion
1. Before exploring the API, use the InfluxDB UI to
```

Reply:
Thanks - debated removing it.